cluster/examples/kubernetes/ceph/monitoring-roles.yaml

```diff
@@ -0,0 +1,79 @@
+#
+# OLM: BEGIN ROLE
+---
+# Aspects for creation of monitoring resources
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: rook-ceph-monitor
+  namespace: rook-ceph
+rules:
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - '*'
+  verbs:
+  - '*'
+---
+# OLM: END ROLE
+#
+# OLM: BEGIN ROLE BINDING
```

Comment:
I think a good rule we should all follow is to keep these headers inside the definition separators for what they are describing. e.g.,
```
---
# OLM: BEGIN description
yaml: stuff
# OLM: END description
---
```

Reply:
Agree. Made changes on my end.